<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color: qlineargradient(spread:pad, x1:0.517523, y1:0, x2:0.506009, y2:1, stop:0 rgba(33, 150, 243, 255), stop:1 rgba(233, 30, 99, 255))

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="memory_text">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
color:  white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_divide">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
        <property name="shortcut">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_clear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>AC</string>
        </property>
        <property name="shortcut">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="shortcut">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="shortcut">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_minus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="shortcut">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_multiply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
        <property name="shortcut">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="shortcut">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="shortcut">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="pushButton_plus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
        <property name="shortcut">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="pushButton_equal">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTipDuration">
         <number>1</number>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButton_dot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
        <property name="shortcut">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" rowspan="3">
       <widget class="QPushButton" name="pushButton_credit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>Credit</string>
        </property>
        <property name="shortcut">
         <string>K</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="shortcut">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QLabel" name="result_text">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
color: white;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
        <property name="shortcut">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_plus_minus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4" rowspan="2">
       <widget class="QPushButton" name="pushButton_backspace">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>&lt;-</string>
        </property>
        <property name="shortcut">
         <string>Backspace</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="shortcut">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
        <property name="shortcut">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="shortcut">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_percent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
        <property name="shortcut">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   border: 0px ;
	border-radius: 12px;
color: white;


}
QPushButton:pressed {
border: 1px solid #ff85e9;
color: white;
   background: transparent;
}

QPushButton:hover {
border: 1px solid #ff85e9;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="shortcut">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
